src/Mod/RuleItem.cpp

```diff
@@ -2664,6 +2682,7 @@ void RuleItem::ScriptRegister(ScriptParserBase* parser)
 	ri.add<&RuleItem::isTwoHanded>("isTwoHanded");
 	ri.add<&RuleItem::isBlockingBothHands>("isBlockingBothHands");
 	ri.add<&isSingleTargetScript>("isSingleTarget");
+	ri.add<&getHaveCategory>("getHaveCategory");
```

Comment:
"getHaveCategory" doesn't sound correct in English. Either "getHasCategory", or even better just "hasCategory"

ri.add<&hasCategoryScript>("hasCategory");